src/main/java/lotto/util/validator/LottoCostValidator.java

```diff
@@ -0,0 +1,20 @@
+package lotto.util.validator;
+
+import lotto.rule.LottoGameRule;
+import lotto.util.exception.ErrorMessage;
+
+public class LottoCostValidator {
+    public static void validateLottoCostUnit(int lottoCost) {
+        if (!(overMinLottoCost(lottoCost) && isZeroRemainder(lottoCost))) {
```

Comment:
가독성을 위해 각 조건들을 메서드로 추출한 건 너무 좋습니다!! 👍 
다만, 조건 메서드에 다시 부정문을 사용해야 한다면, 메서드 자체를 조금 바꿔보는 건 어떨까요?
```suggestion
        if (smallerThanMinLottoCost(lottoCost || isNotZeroRemainder(lottoCost)) {
```

Reply:
안그래도 이 부분은 제가 보기에도 헷갈렸는데😅
차라리 부정 판단 메서드를 만드는게 낫겠군요!